dxr/static/js/dxr.js

```diff
@@ -426,23 +391,30 @@ $(function() {
                     return;
                 }
                 data.query = query;
-                // New results, overwrite
+                // New results, display them.
                 if (myRequestNumber > displayedRequestNumber) {
                     displayedRequestNumber = myRequestNumber;
-                    populateResults(data, false);
-                    var pushHistory = function() {
-                        history.pushState({}, '', queryString);
+                    populateResults(data, appendResults);
+                    var pushHistory = function () {
```

Comment:
They behave the same in this case, so I kept what we had originally. I shall change to the other syntax.